js/editor.js

```diff
@@ -1,6 +1,9 @@
 const { h, render, Component } = preact;
 const { route, Router, Link } = preactRouter;
 
+const REST_URL_temp = 'https://seed-trial.firebaseio.com/user.json';
```

Comment:
This URL should not be in the code.

Reply:
I am so sorry, my bad. :disappointed_relieved: